src/channels/entities/userPrivateChannel.entity.ts

```diff
@@ -0,0 +1,11 @@
+import { UserChannel } from '@prisma/client';
+import { UserEntity } from '../../users/entities/user.entity';
+
+export class UserPrivateChannelEntity implements UserChannel {
```

Comment:
On pourrait faire un extends sur le userChannelEntity

Reply:
Je garde l'idée, mais celà concernerait peut-être une autre branche ? Quand on s'occupera des 1 to 1 ?

Reply:
Ok dans ce cas on peut supprimer ce fichier et on le mettra dans l autre branche

Reply:
Bon, en fait, tant que j'y étais ^^